src/main/java/com/google/devtools/build/lib/worker/WorkerKey.java

```diff
@@ -88,6 +91,10 @@ public boolean mustBeSandboxed() {
     return mustBeSandboxed;
   }
 
+  public boolean proxyed() {
```

Comment:
nit: Non-private symbols require docstrings.

Reply:
I didn't see docstrings for most of the non-private symbols in this file, I was wondering if I should make it an exception when I was working on it. It would be good if the previous authors can add the docstrings as well. I didn't investigate on all of them since I was not using, but I am happy to write for my own one. Thanks.

Reply:
Yeah, it's unfortunate that this code is under-documented and I'm not too happy about it. It's hard to navigate. Any new code should come with docstrings, and it's nice to take the chance to write some while figuring out what a piece of code does if you are already touching it anyway. Thanks.

Reply:
Done.

Reply:
Docstrings added.